rasa/core/actions/action.py

```diff
@@ -930,6 +931,36 @@ async def run(
         return []
 
 
+class ActionDisambiguateFlows(Action):
```

Comment:
should probably be removed, or we can add a default flow like for correction

Reply:
how would accepting work though? 🤔

Reply:
Ah yeah, this was just an early thing and not finished. I'll remove it for now. I was thinking that we should offer buttons for users to pick the option in this case?

Reply:
yes i agree, but my question was more on the underlying system as in how do we use the selected intent 🤔

Reply:
standard intent from regex should work here - button1: /flow1, button2: /flow2

Reply:
ah ok, so actually not correcting but rather sending a new message :+1: yeah that could work

Reply:
it is a little tricky as we'd need to treat the response to this question as the input for the original question

Reply:
Are we on the same page here? This DisambiguateFlows action was originally for asking the user which flow they want to start when there are multiple possible ones. But I haven't really pursued it further. You seem to be talking about the correction action though 🤔

Reply:
I was thinking the disambiguation, at least in the first iteration, could be a flow that has an "action" and not a "question". The action just lists the different options and choosing one of the options starts the new flow. So there wouldn't be a loop involved